modules/global-env.d.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
+/// <reference path="apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts" />
+/// <reference path="apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/index.d.ts" />
+/// <reference path="apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/index.d.ts" />
+/// <reference path="apps/frontend-js/frontend-js-collapse-support-web/src/main/resources/META-INF/resources/index.d.ts" />
+/// <reference path="apps/frontend-js/frontend-js-dropdown-support-web/src/main/resources/META-INF/resources/index.d.ts" />
+/// <reference path="apps/frontend-js/frontend-js-tabs-support-web/src/main/resources/META-INF/resources/index.d.ts" />
```

Comment:
You don't need these here either. The only files you will reference here are files that contribute to the global namespace.